### PR TITLE
Sony web3 incubation program

### PR DESCRIPTION
**Pull request by ayumitk**

### CODE REVIEW

File: lang/ja.json [hunk 1]
```diff
@@ -26,6 +26,11 @@
   },
   "learn_more": "もっと見る",
   "program":{
+    "about":{
```
Comment (ayumitk) [inline lang/ja.json:29]: @Sota20 I added the "About this program" section. Please check these Japanese translations too

File: lang/ja.json [hunk 2]
```diff
@@ -1,77 +1,77 @@
 {
   "meta": {
```
Comment (ayumitk) [inline lang/ja.json:2]: @Sota20 Sorry, I didn't mention you but the "meta" section uses for SEO. Each page's title and description will help users to search in Japanese and find the page. I think we can keep the page title in English, but is it possible to translate the descriptions into Japanese?

File: lang/ja.json (hunk 1 above)
Comment (Sota20) [reply lang/ja.json:29]: Thanks

File: lang/ja.json (hunk 2 above)
Comment (Sota20) [reply lang/ja.json:2]: OK, I translate them

Comment (ayumitk) [reply lang/ja.json:2]: Thank you!

**Review by Sota20 — APPROVED** [review]

**Pull request merged**